extensions/google-workspace/src/open-google-drive-home.tsx

```diff
@@ -0,0 +1,8 @@
+import { withGoogleAuth } from "./components/withGoogleAuth";
+import { openURL } from "./helpers/docs";
+
+async function Command() {
+  await openURL("https://drive.google.com/drive/u/0/my-drive");
+}
+
+export default withGoogleAuth(Command);
```

Comment:
You should add the `authuser` param so that Google Drive opens for the right account: 

```suggestion
import { getUserEmail } from "./api/googleAuth";
import { withGoogleAuth } from "./components/withGoogleAuth";
import { open } from "@raycast/api";

async function Command() {
  const email = await getUserEmail();

  const searchParams = new URLSearchParams();
  if (email) {
    searchParams.append("authuser", email);
  }

  await open(`https://drive.google.com/drive/u/0/my-drive?${searchParams.toString()}`);
}

export default withGoogleAuth(Command);
```

Reply:
Hi, from my previous test, it seems there is no difference in whether or not to add the authuser parameter.

Reply:
There was a difference on my end. It opened Google Drive in my personal account even though I'm logged in with my professional account on Raycast. Adding `authuser` fixes it.